GameFiles/_integration.gsc

```diff
@@ -633,14 +699,22 @@ ParseDataString( data )
     dict = [];
     
     counter = 0;
-    foreach ( part in dataParts )
+    if( isdefined( dataParts ) )
     {
-        splitPart = strtok( part, "=" );
-        key = splitPart[0];
-        value = splitPart[1];
-        dict[key] = value;
-        dict[counter] = key;
-        counter++;
+        //foreach ( part in dataParts )
+        for( i = 0; i < dataParts.size; i++ )
```

Comment:
How come foreach doesn't work here? can we remove the commented code if it's not usable

Reply:
T5 doesn't know the foreach keyword.